src/tools/gxhttpd/opt.ss

```diff
@@ -0,0 +1,119 @@
+;;; -*- Gerbil -*-
+;;; © vyzo
+;;; The Gerbil HTTP Daemon
+;;;
+(import :std/cli/getopt
+        :std/config)
+(export #t)
+
+(def server-config-option
+  (option 'config "-c" "--config"
+    help: "location of the httpd configuration; when running as a standalone server it defaults to $GERBIL_PATH/httpd/config; when running as part of an ensemble this option is ignored"))
+
+(def ensemble-config-option
+  (option 'config "-c" "--config"
+    help: "location of the httpd ensemble configuration; it defaults to $GERBIL_PATH/ensemble/config"))
+
+(def server-id-optional-argument
+  (optional-argument 'server-id
+    value: string->object
+    help: "when running as part of an ensemble, this is the ensemble server id"))
+
+(def server-cmd
+  (command 'server
+    server-config-option
+    server-id-optional-argument
+    help: "runs a single httpd server"))
+
+(def ensemble-cmd
+  (command 'ensemble
+    ensemble-config-option
+    help: "runs a supervied httpd server ensemble"))
+
+(def config-ensemble-flag
+  (flag 'ensemble "--ensemble"
+    help: "configure the httpd ensemble"))
+
+(def config-print-flag
+  (flag 'print "-p" "--print"
+    help: "print the configuration"))
+
+(def config-set-flag
+  (flag 'set "--set"
+    help: "override the configuration instead of merging"))
+
+(def config-path-option
+  (option 'config "-c" "--config"
+    help: "specify the configuration path"))
```

Comment:
Another "-c" here.

Reply:
yeah, i guess i could compress to just this.